templates/home/index.html

```diff
@@ -135,131 +135,40 @@ <h1>Our Initiatives</h1>
             <div class="glide">
                   <div class="glide__track" data-glide-el="track">
                     <ul class="glide__slides">
+                        
+                        {% for initiative in initiatives %}
+
                           <li class="glide__slide">
                               <div class="card">
                                 <div class="card-img">
-                                    <img src='{% static "home/images/02.jpg" %}'>
+                                  {% if not initiative.thumbnail %}  
```

Comment:
I think `not` shouldn't be there in the first statement if there's an other option because using `not` generally creates confusion.